src/Storage/Storage/Blob/Cmdlet/NewAzureStorageContainerSasToken.cs

```diff
@@ -85,7 +89,14 @@ public string Policy
         public override int? ServerTimeoutPerRequest { get; set; }
         public override int? ClientTimeoutPerRequest { get; set; }
         public override int? ConcurrentTaskCount { get; set; }
-
+        protected override bool UseTrack2SDK()
```

Comment:
Do you want to rename this function to `UseTrack2Sdk`?

Reply:
updated